src/parsers/power_system_table_data.jl

```diff
@@ -1195,6 +1195,7 @@ function get_storage_by_generator(data::PowerSystemTableData, gen_name::Abstract
     end
 
     if length(head) != 1
+        @show head
```

Comment:
Why is this in here?

Reply:
accidental addition. I'll delete it. Thanks.